README.md

```diff
@@ -37,7 +37,9 @@ Change summary
 | Flatline   | 30 -> 20 (4 -> 5 BTK) | 20 (5 BTK) (_vanilla_) | 15 (7 BTK) (_vanilla_) |
 | Spitfire   | 35 -> 25 (3 -> 4 BTK) | 25 -> 20 (4 -> 5 BTK)  | 20 -> 17 (5 -> 6 BTK)  |
 | EVA-8      | 200 -> 150            | 10 (_vanilla_)         | -                      |
-| Melee      | 100 -> 70             | -                      | -                      |
+| Melee      | 100 -> 70             | -                      | -                      | 
+| Re45       | 20 -> 15 (4 -> 7 BTK) | 13 -> 12 (7 -> 9  BTK) | 9  -> 8 (12 -> 13 BTK) |
```

Comment:
Any reason why near damage have +3 BTK, far damage +2 BTK and very far damage +1 BTK, while P20 has +1 BTK for every range?

Reply:
rate of fire boss. while p2016 is still spammy its nowhere as fast as a burst from an re45. 
as for the declining nerf for re45. i didnt want it abused but should still be able to kill at the very least as a finisher of sorts for those 1 hp escapees instead of "man i cant hit with a projectile, welp time to go re45"

Reply:
good point

Reply:
Actually yeah this feels better than my idea after testing on marvins 👍